docs/indexOLD.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
If we don't need this file any more, we should just delete it rather than rename to `indexOLD`

Reply:
Done